### PR TITLE
KiwiForGmail: Add version 2.0.480

### PR DESCRIPTION
**Pull request by r15ch13**

### CODE REVIEW

File: bucket/kiwiforgmail.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://s3-us-west-1.amazonaws.com/kiwiforgmail/windows/release/64bit/KiwiForGmail-2.0.480-full.nupkg",
+            "hash": "sha1:73681e27dd38c046bd2bc0e673b792ca9efd9066"
+        },
+        "32bit": {
+            "url": "https://s3-us-west-1.amazonaws.com/kiwiforgmail/windows/release/32bit/KiwiForGmail-2.0.480-full.nupkg",
+            "hash": "sha1:7929b279021dc778a131f63d1cad90cf32da3268"
+        }
+    },
```
Comment (Ash258) [inline bucket/kiwiforgmail.json]:
```suggestion
    },
    "extract_dir": "lib\\net45",
```

File: bucket/kiwiforgmail.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://kiwiforgmail.com/",
+    "description": "A desktop client for Gmail annd Google Apps.",
+    "license": "Proprietary",
+    "version": "2.0.480",
+    "extract_dir": "lib\\net45",
```
Comment (Ash258) [inline bucket/kiwiforgmail.json]:
```suggestion
```

File: bucket/kiwiforgmail.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://kiwiforgmail.com/",
+    "description": "A desktop client for Gmail annd Google Apps.",
```
Comment (Ash258) [inline bucket/kiwiforgmail.json]:
```suggestion
    "description": "A desktop client for Gmail annd Google Apps",
```

Comment (linsui) [reply bucket/kiwiforgmail.json]: `annd`? I guess it's a typo.

**Pull request merged**